gravitee-apim-distribution/pom.xml

```diff
@@ -1409,6 +1409,19 @@
                         </exclusion>
                     </exclusions>
                 </dependency>
+                <dependency>
+                    <groupId>com.graviteesource.reactor</groupId>
+                    <artifactId>gravitee-reactor-native-kafka</artifactId>
+                    <version>${gravitee-reactor-native-kafka.version}</version>
+                    <type>zip</type>
+                    <scope>runtime</scope>
+                    <exclusions>
+                        <exclusion>
+                            <groupId>*</groupId>
+                            <artifactId>*</artifactId>
+                        </exclusion>
+                    </exclusions>
+                </dependency>
```

Comment:
You have added it to default-bundle. Is it something we want ? Or do we want users to add this reactor separately ?

Reply:
did the same as for message-reactor, I will ask Anthony

Reply:
as discussed with @anthonyjgatti we can keep it in default bundle